internal/kallsyms/kallsyms.go

```diff
@@ -52,6 +55,8 @@ func FlushKernelModuleCache() {
 	kernelModules.kmods = nil
 }
 
+var ErrKsymIsUnambiguous = errors.New("ksym is unambiguous")
```

Comment:
I think this should read `ksym is ambiguous`.

This sentinel is only checked from within the package itself, there's no need for this to be exported. An external caller wouldn't need to react to it at runtime, for example. Maybe they could, but it would involve trying to load a different ELF that doesn't refer to the symbol. If this ever becomes relevant, it can be exported, but unexporting a symbol users depend on is more difficult to justify.